docs/basic_filtering.rst

```diff
@@ -38,6 +38,10 @@ The following table provides an overview of these different grid type options: w
 | ``IRREGULAR_WITH_LAND``        | locally orthogonal grid                                         | yes          | periodic           | Scalar Laplacian | :doc:`examples/example_filter_types`;    |
 |                                |                                                                 |              |                    |                  | :doc:`examples/example_tripole_grid`     |
 +--------------------------------+-----------------------------------------------------------------+--------------+--------------------+------------------+------------------------------------------+
+| ``MOM5U``                      | Velocity-point on Arakawa B-Grid                                | yes          | tripole            | Scalar Laplacian |                                          |
```

Comment:
As of now, the Laplacians `MOM5U` and `MOM5T` do not use a tripole boundary condition (even though this could be easily done following the implementation for the `TRIPOLAR_POP_WITH_LAND` Laplacian). I think we should change `tripole` --> `periodic`.